knowledge-base/set-culture-of-date-picker-in-angular-viewer.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Change the Culture of Kendo DatePicker Widget in the Parameters Area of the Angular Report Viewer
+description: How to Set Culture for Kendo Widgets used in Angular Report Viewer
+type: how-to
+page_title: Localize the Angular Viewer Kendo Widgets
+slug: set-culture-of-date-picker-in-angular-viewer
+position: 
+tags: 
+ticketid: 1459450
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+The article describes how to change the culture of the Kendo widgets used to build the Angular Report Viewer.
+
+## Solution
+The general solution of the problem for localizing the DatePicker of the 
+[pure Html5 Report Viewer](https://docs.telerik.com/reporting/html5-report-viewer) is described in the 
+[Change the culture of Kendo DatePicker widget in the parameters area of the HTML5 Report Viewer](./how-to-change-the-culture-of-the-datepicker-in-html5-report-viewer) 
+article. The Angular viewer is a wrapper of the pure Html5 viewer that utilizes Kendo UI for jQuery components. For that reason, 
+in order to update the DatePicker component of the Angular Viewer according to the above article, you need first to integrate the 
+Kendo jQuery components in the Angular application following the article 
+[Kendo UI for jQuery Integration](https://www.telerik.com/kendo-angular-ui/components/framework/kendo-jquery/). Here are the steps: 
+
+- Install Kendo UI for jQuery:
+```
+npm install --save @progress/kendo-ui
+``` 
+
+ - In the 'app.component.ts' import the components from Kendo and declare the 'kendo' object. Then in the 'AfterViewInit' add 
```

Comment:
I could not remember wheel at the moment where exactly we place this in our demo project, but based on the kendo article, the import of the kendo-ui should be in the app module, not app.component.ts . Check this info once again.